<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrdenarArreglo</class>
 <widget class="QDialog" name="OrdenarArreglo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushOrdenar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ordenar</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>31</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>31</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="agregarArray">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinOrdenar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
